src/index.js

```diff
@@ -21,15 +21,24 @@ function adapter() {
   }.bind(this);
 }
 
-function reset() {
-  this.handlers = VERBS.reduce(function(accumulator, verb) {
+function getVerbObject() {
+  return VERBS.reduce(function(accumulator, verb) {
     accumulator[verb] = [];
     return accumulator;
   }, {});
 }
 
+function reset() {
+  this.handlers = getVerbObject();
```

Comment:
Should `reset` also reset history?

Reply:
I would be open to that. If the `reset` function reset both history and handlers, and there is already a separate `resetHistory` function, would it also make sense to have a `resetHandlers` function?

Reply:
Yeah, that would make sense.